modules/gumgumBidAdapter.js

```diff
@@ -247,6 +247,39 @@ function _getFloor(mediaTypes, staticBidFloor, bid) {
   return bidFloor;
 }
 
+/**
+ * Retrieves the device data from the ORTB2 object
+ * @param {Object} ortb2Data ORTB2 object
+ * @returns {Object} Device data
+ */
+function _getDeviceData(ortb2Data) {
+  const _device = deepAccess(ortb2Data, 'device') || {};
+
+  // set device data params from ortb2
+  const _deviceRequestParams = {
```

Comment:
@BohdanVV can you add ip and ipv6 to this?

Reply:
> @BohdanVV can you add ip and ipv6 to this?

Done 👌🏻